legend-sdlc-server/src/main/java/org/finos/legend/sdlc/server/gitlab/api/GitLabPatchApi.java

```diff
@@ -224,6 +224,34 @@ else if (maxPatchVersion != null)
         }
     }
 
+    @Override
+    public Patch getPatch(String projectId, VersionId patchReleaseVersionId)
+    {
+        LegendSDLCServerException.validateNonNull(projectId, "projectId may not be null");
+        LegendSDLCServerException.validateNonNull(patchReleaseVersionId, "patchReleaseVersionId may not be null");
+
+        GitLabProjectId gitLabProjectId = parseProjectId(projectId);
+        String branchName = getPatchReleaseBranchName(patchReleaseVersionId);
+
+        // Get patch branch
+        try
+        {
+            Pager<Branch> pager = getGitLabApi().getRepositoryApi().getBranches(gitLabProjectId.getGitLabId(), "^" + branchName, ITEMS_PER_PAGE);
+            Stream<Patch> stream = PagerTools.stream(pager)
+                    .filter(branch -> (branch != null) && (branch.getName() != null) && branch.getName().equals(branchName))
+                    .map(branch -> fromPatchBranchName(projectId, branch.getName()));
+           Patch currentPatch = stream.collect(Collectors.toList()).get(0);
+           return currentPatch;
```

Comment:
Since this is looking for a single branch, it should probably use the getBranch method instead of getBranches. I would expect this implementation to look a lot like GitLabWorkspaceApi.getWorkspace.